scripts/run-encryption-tests-local.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# sets up an encrypted mongodb cluster, adds relevant variables to the environment, and runs encryption tests
+
+export CWD=$(pwd);
+
+# set up encrypted mongodb cluster if the encrypted-cluster folder does not exist
+# note: for tooling, cluster set-up and configuration look into the 'scripts/start-encrypted-cluster.sh' script
+if [ -d "encrypted-cluster" ]; then
+  cd encrypted-cluster
+else
+  source $CWD/scripts/start-encrypted-cluster.sh
+fi
+
+# extracts MONGOOSE_TEST_URI and CRYPT_SHARED_LIB_PATH from .yml file into environment variables for this test run
```

Comment:
Not quite - this script doesn't load the files into the environment.  Instead, it reformats the yaml file into bash syntax (`export VAR=<value>`).  the variables are loaded into the environment when we call `source expansions.sh`

Reply:
Sure, that's fine too.  I just wanted to clarify what the script does.